packages/core/admin/admin/src/content-manager/pages/App/reducer.js

```diff
@@ -20,14 +20,14 @@ const initialState = {
 const mainReducer = (state = initialState, action) =>
   produce(state, (draftState) => {
     switch (action.type) {
-      case GET_DATA: {
+      case GET_INIT_DATA: {
         draftState.status = 'loading';
         break;
       }
```

Comment:
I'm not sure I understand? Shouldn't it be SET_LOADING_STATUS?

Reply:
I think the way to read it is that we're initiating a fetch, which is why it triggers a loading state.

Your approach works too, but since the GET_DATA pattern is used in other reducers, I'd prefer to keep it for consistency